llama_stack/apis/providers/providers.py

```diff
@@ -19,6 +19,17 @@ class ProviderInfo(BaseModel):
     provider_type: str
     config: dict[str, Any]
     health: HealthResponse
+    metrics: str | None = None  # define as string type than httpurl for openapi compatibility
```

Comment:
Please use a `Field` and add a description to it. I know other fields don't but we need to be better at this. Thanks

Reply:
probably better to keep it to str for now.
if we wanna make it httpurl later, can update make schema more flex for types first.